alembic/versions/c6250555a36c_.py

```diff
@@ -0,0 +1,370 @@
+"""
+
+Revision ID: c6250555a36c
+Revises: d7c2f99cd14d
+Create Date: 2020-04-06 09:34:50.929724
+
+"""
+import enum
+from datetime import datetime, timezone
+from json import loads
+from os import getenv
+from copr.v3 import CoprNoResultException, Client
+from celery.backends.database import Task
+from redis import Redis
+from typing import TYPE_CHECKING, Union
+from persistentdict.dict_in_redis import PersistentDict
+
+from alembic import op
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    DateTime,
+    ForeignKey,
+    Enum,
+    orm,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session, relationship
+from sqlalchemy.types import PickleType
+
+from packit_service.service.models import Installation, CoprBuild
```

Comment:
Please, copy also these models to be sure we are using the right version. (Some skeleton should be enough.)